src/slider.styles.ts

```diff
@@ -0,0 +1,174 @@
+      /* stylelint-disable-next-line property-no-vendor-prefix */
+      -moz-appearance: textfield;
+      background-color: var(--glide-core-color-interactive-surface-container);
+      block-size: 2.125rem;
+      border: 1px solid var(--glide-core-color-interactive-stroke-primary);
+      border-radius: var(--glide-core-rounding-base-radius-sm);
+      box-sizing: border-box;
+      inline-size: 2.125rem;
+      text-align: center;
+
+      &::-webkit-outer-spin-button,
+      &::-webkit-inner-spin-button {
+        appearance: none;
+      }
+
+      &:focus,
+      &:hover {
+        border-color: var(--glide-core-color-interactive-stroke-primary--hover);
+        outline: none;
+        transition: border-color 200ms ease-in-out;
+      }
+
+      &.disabled {
+        background-color: var(
+          --glide-core-color-interactive-surface-container--disabled
+        );
+        border-color: var(
+          --glide-core-color-interactive-stroke-primary--disabled
+        );
+        color: var(--glide-core-color-interactive-text-default--disabled);
+      }
+
+      &.error {
+        border-color: var(--glide-core-color-advisory-stroke-error-primary);
+      }
+
+      /*
+        We had to resort to a class selector because there may be a bug in Chrome and Safari
+        with ":read-only": https://bugs.chromium.org/p/chromium/issues/detail?id=1519649
+      */
+      &.readonly {
+        background-color: transparent;
+        border: 1px solid transparent;
+        padding-inline-start: 0;
+      }
+    }
+
+    .track-container {
+      flex: 1;
+      min-inline-size: 0;
+
+      &.single {
+        /*
+            Without this suggestion from design, when the handle
+            is at the min, it would overflow the container and
+            not align with label. This isn't needed in multiple
+            mode because we have an input to help prevent this
+            from happening.
+         */
+        padding-inline-start: var(--glide-core-spacing-base-sm);
+      }
+    }
+
+    .filled-track {
+      background-color: var(
+        --glide-core-color-interactive-surface-container-active
+      );
+      block-size: var(--private-track-size);
+      border-radius: var(--glide-core-rounding-base-radius-round);
+
+      /*
+        Absolute positioning is required here, as JavaScript is
+        used to fill it as the user interacts with the component.
+      */
+      position: absolute;
+
+      &.disabled {
+        background-color: var(
+          --glide-core-color-interactive-surface-container-inactive
+        );
+      }
+    }
+
+    .unfilled-track {
+      background-color: var(--glide-core-color-static-stroke-primary);
+      block-size: var(--private-track-size);
+      border-radius: var(--glide-core-rounding-base-radius-round);
+      inline-size: 100%;
+      position: relative;
+
+      &.disabled {
+        background-color: var(--glide-core-color-static-stroke-secondary);
+      }
+    }
+
+    .handle {
```

Comment:
Didn't like `.thumb`?

Reply:
I actually had it as `.thumb` since the beginning, as that's what the platform calls it. But design refers to it as a "handle", so I opted for that instead.  I can ask if they'd be open to calling it a thumb instead!

Reply:
Nice. Chatted with Zheng about it, he's open to switching it back to thumb if we feel strongly about it. But I'm fine with "handle", only because we both (design + engineering) are already calling it that at this point.